.circleci/config.yml

```diff
@@ -79,6 +79,11 @@ reference:
       name: Decode Android key store
       command: echo $ANDROID_KEYSTORE | base64 -di > keystore.jks
 
+  export_github_release_token: &export_github_release_token
+    run:
+      name: Export Github release key
+      command echo 'export GITHUB_RELEASE_TOKEN="$GITHUB_RELEASE_TOKEN"' >> $BASH_ENV
```

Comment:
I think this is missing a colon at the end of `command` which is leading to CircleCI not building

Reply:
Yep, you were right